lib/feedbackMessages.js

```diff
@@ -0,0 +1,25 @@
+class FeedbackMessages {
+  constructor(user, surveyLink) {
+    this.user = user;
+    this.surveyLink = surveyLink;
+  }
+}
+
+class IssueFeedback extends FeedbackMessages {
+  constructor(user, surveyLink) {
+    super(user, surveyLink);
+  }
+  success(msg) {
+    return `# Step feedback for ${this.user}\n${msg}** task!\n\n_please [provide feedback](${this.surveyLink}) for this lab_`;
+  }
+
+  failure(err) {
+    return `# ${this.user} It looks like you have an error 😦\n**We expected:**\n ${err.expected}\n**We received:**\n ${err.got}`;
+  }
+
+  error(err, payload) {
+    return `# ${err.name}\n${err.userMessage}\n**payload details:**\n\`\`\`${payload.err}\`\`\``;
+  }
```

Comment:
@hectorsector This is the spot I can use some help on.  The goal here is to standardize some feedback messaging.  You can use Markdown characters.  These strings just become the body of the issues when issues are the desired feedback type.

**success:** What to tell the learner when they get things right
**failure:** What to tell the learner when they get things wrong
**error:** What to tell the learner when something breaks that it outside of their control, like the API being offline.

Reply:
You can even do this the easy way and write regular markdown as comments in this PR and I'll translate it to fit inside these methods 😄